### PR TITLE
Wish-list & Cart Management

### PR DESCRIPTION
**Pull request by MarkVed17**
~ Added wish-list page
~ Added cart-management page
~ Here are the previews:
https://deploy-preview-3--campcretaceous-screens.netlify.app/wishlist/wishlist-page.html
https://deploy-preview-3--campcretaceous-screens.netlify.app/cart-management/cart-page.html

### CODE REVIEW

File: cart-management/cart-page.css [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  gap: 1rem;
+}
+
+.cart-heading {
+  flex-basis: 100%;
+}
+
+.cart-item {
+  display: flex;
+  /* width: 80%; */
```
Comment (horsemaker) [inline cart-management/cart-page.css:29]: If this comment is not intentional then do remove it.

File: cart-management/cart-page.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  background-color: var(--card-bg-color);
+  height: fit-content;
+  padding: 1.5rem 0;
+  box-shadow: rgba(0, 0, 0, 0.5) 0px 4px 12px;
+}
+
+.cart-summary > h1 {
+  margin: 0;
+  /* padding: 1rem; */
+  /* padding-top: 1rem; */
```
Comment (horsemaker) [inline cart-management/cart-page.css:110]: If not needed then do remove these comments too.

**Review by Amandeep-sinha — APPROVED** [review]
looks good to go ahead

**Review by horsemaker — APPROVED** [review]
Good use of comments in the HTML files to distinguish components.

**Review by suryxks — APPROVED** [review]
you can remove unused code comments 
use varables for box shadow if you are using it in more than one place

**Review by palash23jain — APPROVED** [review]
you can use color variables consistently instead of using at some places. Looks good to go

**Pull request merged**